apps/docs/src/app/core/component-docs/dynamic-side-content/dynamic-side-content-docs.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+
+import * as basicExampleHtml from '!raw-loader!./examples/dynamic-side-content-basic-example.component.html';
+import * as positioningExampleHtml from '!raw-loader!./examples/dynamic-side-content-positioning-example.component.html';
+import * as sizeExampleHtml from '!raw-loader!./examples/dynamic-side-content-size-example.component.html';
+
+@Component({
+    templateUrl: './dynamic-side-content-docs.component.html'
+})
+export class DynamicSideContentDocsComponent {
+    basic: ExampleFile[] = [
+        {
+            language: 'html',
+            code: basicExampleHtml,
+            fileName: 'dynamic-side-content-basic-example'
+        }
+    ];
+
+    positioning: ExampleFile[] = [
+        {
+            language: 'html',
+            code: positioningExampleHtml,
+            fileName: 'dynamic-side-content-positioning-example'
+        }
+    ];
+
+    size: ExampleFile[] = [
+        {
+            language: 'html',
+            code: sizeExampleHtml,
+            fileName: 'dynamic-side-content-size-example'
+        }
+    ];
```

Comment:
Could you add scss files code to those examples? Like [here](https://github.com/SAP/fundamental-ngx/wiki/Stackblitz-support-instructions#only-html-document-is-displayed-and-it-has-its-own-ts-file-new-if-there-is-anything-in-ts-file-instead-of-empty-file), just to make sure stackblitz will work

Reply:
fixed